packages/fxa-settings/src/pages/Signin/SigninTokenCode/container.tsx

```diff
@@ -35,20 +35,22 @@ const SigninTokenCodeContainer = ({
     integration
   );
 
-  // Note the exception to using `useAccount` hook. We needed to do this because
-  // using the gql api was failing to retrieve the totp status with `Must verify`
-  // error.
   const [totpEnabled, setTotpEnabled] = useState<boolean>(false);
-  const session = useSession();
   useEffect(() => {
+    if (!signinState) {
```

Comment:
Should this be `if (!signinState || !signinState.sessionToken) {`, so that it's symmetric with case below. If there's not a session token, I'm not sure why we'd continue and call getTotpStatus.